spec/polaris-management-service.yml

```diff
@@ -901,6 +903,58 @@ components:
       required:
         - roleArn
 
+    S3StorageConfigInfo:
+      type: object
+      description: S3 compatible storage configuration info (MinIO, Dell ECS, Netapp StorageGRID, ...)
+      allOf:
+        - $ref: '#/components/schemas/StorageConfigInfo'
+      properties:
+        credsCatalogAndClientStrategy:
+          type: string
+          enum:
+            - VALUE
+            - ENV_VAR_NAME
+          default: ENV_VAR_NAME
+          example: "ACCESS_KEY"
+          description: When you send key VALUE directly via this command, they should apear in logs. By ENV_VAR_NAME without dollar, only a reference will appear in logs, but the value have to be available as environnement variable in the context where Polaris is running
+        credsVendingStrategy:
+          type: string
+          enum:
+            - TOKEN_WITH_ASSUME_ROLE
+            - KEYS_SAME_AS_CATALOG
```

Comment:
It seems like `KEYS_SAME_AS_CATALOG` is the same as just disabling subscoping, is that right?

Reply:
Exactly, explained like that it's much clearer.

I was trying also to do a distinction with the third option where it is possible to use another pair of keys, not the one used by the catalog.

Reply:
Maybe instead of this, we should expect the user to just disable subscoping via `SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION`?

Reply:
Correct. Default value False. With a clear description to explain users not familiar with concept of subScoping" implemented in Polaris. I will refactor this way.